includes/connect-existing-pages.php

```diff
@@ -45,6 +45,10 @@ function wc_admin_get_core_pages_to_connect() {
 				)
 			),
 		),
+		'wc-subscriptions' => array(
+			'title' => __( 'My Subscriptions', 'woocommerce-admin' ),
+			'tabs'  => array(),
+		),
```

Comment:
Do we still need this, since we're using `wc-addons` for both Marketplace and My Subscriptions?